### PR TITLE
KEYCLOAK-14178 Avoid erroneous HTML escaping in KeycloakServer

### PR DESCRIPTION
**Pull request by thomasdarimont**
This PR disables the Resteasy HTML Sanitizer in KeycloakServer.
Previously KeycloakTest server erroneously escapes html in error pages.

See the linked JIRA issue for details.

### CODE REVIEW

**Review by mhajas — APPROVED** [review]
Thank you very much @thomasdarimont! This fixes the issue. More details in https://issues.redhat.com/browse/KEYCLOAK-14178 comments.

File: testsuite/utils/src/main/java/org/keycloak/testsuite/KeycloakServer.java [hunk 1]
```diff
@@ -400,15 +401,11 @@ public void start() throws Throwable {
 
             di.setDefaultServletConfig(new DefaultServletConfig(true));
 
-            ServletInfo restEasyDispatcher = Servlets.servlet("Keycloak REST Interface", HttpServlet30Dispatcher.class);
-
-            restEasyDispatcher.addInitParam("resteasy.servlet.mapping.prefix", "/");
-            restEasyDispatcher.setAsyncSupported(true);
-
-            di.addServlet(restEasyDispatcher);
+            // Note that the ResteasyServlet is configured via server.undertowDeployment(...);
```
Comment (hmlnarik) [inline testsuite/utils/src/main/java/org/keycloak/testsuite/KeycloakServer.java:404]: @thomasdarimont Are `asyncSupported` and `resteasy.servlet.mapping.prefix` parameter maintained? Perhaps they need to be added similarly as a property of the `deployment`?

Comment (thomasdarimont) [reply testsuite/utils/src/main/java/org/keycloak/testsuite/KeycloakServer.java:404]: IMHO yes: https://github.com/resteasy/Resteasy/blob/master/server-adapters/resteasy-undertow/src/main/java/org/jboss/resteasy/plugins/server/undertow/UndertowJaxrsServer.java#L184

Comment (hmlnarik) [reply testsuite/utils/src/main/java/org/keycloak/testsuite/KeycloakServer.java:404]: Perfect, thanks for checking!

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**